### ISSUE TITLE
Problems inputting PF candidate array to fastjet clustering

### ISSUE THREAD

**Issue by jennetd** [scikit-hep/fastjet#298]
I have a dask_awkward array of particle flow candidates that gives an error when I try to run jet clustering using fastjet. A minimal example is here:
https://github.com/jennetd/hbb-run3/blob/master/data-mc/MinimalExample.ipynb

I am using the newly created image coffeateam/coffea-dask-almalinux8:latest, which includes the following:

```
>>> fastjet.__version__
'3.4.1.3'
>>> awkward.__version__
'2.6.4'
>>> dask_awkward.__version__
'2024.3.0'
```

Please let me know if you have any insights, or if I can provide any more information.

Jennet

**Comment by jennetd** [comment]
I think this error is resulting from the behavior on arrays containing Nones

**Comment by lgray** [comment]
Can you `ak.drop_none` and it suddenly works?

**Comment by jennetd** [comment]
Not exactly. If I flatten the pf candidates it works. I updated the minimal example to show this
